pkg/manifests/rte/rte.go

```diff
@@ -165,25 +197,33 @@ func (mf Manifests) ToDeletableObjects(hp *deployer.Helper, log tlog.Logger) []d
 	return objs
 }
 
-func New(plat platform.Platform) Manifests {
-	mf := Manifests{
+func New(plat platform.Platform) *Manifests {
```

Comment:
I'm still not sold about the pointer. The intent of the previous function was indeed to suggest the the functions were not changing the object - this is why the DeepCopy() and friends.

Reply:
revert it back

Reply:
Under additional thinking, let leave this one with a pointer to avoid the creation of the new object during the assignment.

Reply:
I prefer to rever this change, please. If we do this change, and that will require discussion, we will need to update the other subpackages (`pkg/manifests/{sched,api}`) to have a consistent codebase.